deck.py

```diff
@@ -0,0 +1,87 @@
+from card import Card
+from card import NormalCard
+from card import SpecialCard
+from random import randint
+
+
+class Deck:
+    def __init__(self, cards):
+        self.cards = cards
```

Comment:
We don't want to give user opportunity to change our cards when object is created so make it private

Reply:
I mean make cards private, not class :)

```suggestion
        self._cards = cards
```

Reply:
My bad